src/Components/v2/__tests__/SelectedCareerAchievements.test.tsx

```diff
@@ -40,4 +40,29 @@ describe("SelectedCareerAchievements", () => {
     expect(text).toContain("Reviewed by a major art publication")
     expect(text).toContain("Participated in a major biennial")
   })
+
+  it("renders selected career achievements if no auction results or partner highlights", async () => {
+    ArtistFixture.auctionResults = null
+    ArtistFixture.highlights.partners = null
+    wrapper = await getWrapper(ArtistFixture)
```

Comment:
For good data hygiene we should avoid mutating the `ArtistFixture` in these tests. Although the mutations can't escape outside of the bounds of this file (because of the way jest executes test files in isolation) it could cause confusing problems if people add or modify tests here in the future.

The idiomatic and safe thing to do would be to construct a new version of the artist fixture using object spread syntax, e.g.

```ts
wrapper = await getWrapper({
   ...ArtistFixture,
   auctionResults: null,
   highlights: {
    ...ArtistFixture.highlights,
    partners: null
   }
})
```